src/app/user/shared/component/pre/pre.component.ts

```diff
@@ -0,0 +1,24 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+
+@Component({
+  selector: 'app-pre',
+  templateUrl: './pre.component.html',
+  styleUrls: ['./pre.component.scss']
+})
+export class PreComponent implements OnInit {
+  prev: number;
+
+  constructor(private activatedRoute: ActivatedRoute) {
+  }
+
+  ngOnInit() {
+    this.activatedRoute.url.subscribe(
+      urlSegments => {
+        console.log(`Current Url Query ${urlSegments[1].path}`);
+        this.prev = +urlSegments[1].path - 1;
```

Comment:
for now only I will accept this, keep this n mind and create ticket for future enhancements on asana